rdr_service/dao/genomics_dao.py

```diff
@@ -1239,6 +1239,25 @@ def update(self, obj):
     def _is_valid_set_member_job_field(cls, job_field_name):
         return job_field_name is not None and hasattr(GenomicSetMember, job_field_name)
 
+    def get_members_from_biobank_ids(self, biobank_ids, genome_type=None):
+        """
+        Returns genomicSetMember objects for list of Biobank IDs.
+        :param biobank_ids:
+        :param genome_type:
+        :return:
+        """
+        with self.session() as session:
+            members = session.query(GenomicSetMember).filter(
+                GenomicSetMember.biobankId.in_(biobank_ids),
+                GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE,
```

Comment:
You'll want to check `GenomicSetMember.ignore_flag == 0` here too.

Reply:
Does GenomicSetMemberDao.get_members_from_sample_ids need this added to the filter as well?

Reply:
Yea, we probably should add it. If it doesn't affect this PR, we can have it as a todo.